tester_app/client/app/bundles/HelloWorld/startup/clientGlobals.jsx

```diff
@@ -0,0 +1,3 @@
+import helloWorldClientEntryComponent from './helloWorldClientEntryComponent';
+
```

Comment:
`startup/HelloWorldApp` :+1: 
`containers/HelloWorld` :+1: 
`components/HelloWorldWidget` -> `components/HelloWorld` in the real world we have tons of components, so I'd strip out `Widget` suffix.